playbooks/besman-sast-sonarqube-playbook.sh

```diff
@@ -0,0 +1,151 @@
+#!/bin/bash
+
+# Author: Samir Ranjan Parhi
+# License: Same as Repository Licence
+# Usage : Beta-Release
+# Date : 12/04/2024
+
+function __besman_init() {
+    __besman_echo_white "initialising"
+    export ASSESSMENT_TOOL_NAME="sonarqube"
+    export ASSESSMENT_TOOL_TYPE="SAST"
+    export ASSESSMENT_TOOL_VERSION="9.9.4"
+    export ASSESSMENT_TOOL_PLAYBOOK="besman-$ASSESSMENT_TOOL_TYPE-$ASSESSMENT_TOOL_NAME-playbook.sh"
+    
+    export steps_file_name="besman-$ASSESSMENT_TOOL_TYPE-$ASSESSMENT_TOOL_NAME-steps.md"
+    export BESMAN_STEPS_FILE_PATH="$BESMAN_PLAYBOOK_DIR/$steps_file_name"
+
+
```

Comment:
There was a var array here, and a condition to check if any of the required vars are empty. Please add that as well.

Reply:
@asa1997 : I am Sorry, But I will not do any changes unless the above query are addressed